Prace_domowe/Praca_domowa2/Grupa4/Mateusz Stączek/notebook.ipynb

```diff
@@ -0,0 +1,801 @@
+{
```

Comment:
<p>Count Encoder - no zasadniczo kiedy liczność&nbsp;jakoś odpowiada targetowi. Wiemy, że jeśli czegoś&nbsp;jest dużo, to jest duże</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/75/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>